DigitalLearningSolutions.Data/Services/SectionContentService.cs

```diff
@@ -128,8 +128,14 @@ GROUP BY
                     if (sectionContent == null)
                     {
                         sectionContent = section;
+                        if (!section.DiagnosticStatus)
```

Comment:
Hm, I'd prefer to avoid this nested if statement. I think I'd prefer using ternary operators:
```
sectionContent.DiagnosticAttempts = section.DiagnosticStatus ? sectionContent.DiagnosticAttempts : 0;
...
```
What do you think? 🙂

Reply:
Yeah when I wrote this I was trying to think of a better way to do it. I thought that ternary operators would be too much repetition of the same thing and it felt less clear that the 3 statements were linked by the same condition. I'm happy to change to ternary operators if you think that's better?

Reply:
Yes I think this is a case where neither is ideal and it's more a case if personal preference. I think I'd find the ternary operators slightly more readable 🙂